### PR TITLE
add ratelimit populate descriptors log

### PR DESCRIPTION
**Pull request by zirain**
Signed-off-by: hejianpeng <hejianpeng2@huawei.com>

Commit Message: add ratelimit populate descriptors log to help debug
Additional Description:
Risk Level: low
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

File: source/extensions/filters/http/local_ratelimit/local_ratelimit.cc [hunk 1]
```diff
@@ -91,6 +91,12 @@ Http::FilterHeadersStatus Filter::decodeHeaders(Http::RequestHeaderMap& headers,
     populateDescriptors(descriptors, headers);
   }
 
+  for (const auto& request_descriptor : descriptors) {
+    for (const Envoy::RateLimit::DescriptorEntry& entry : request_descriptor.entries_) {
+      ENVOY_LOG(debug, "populate descriptors: key={} value={}", entry.key_, entry.value_);
```
Comment (daixiang0) [inline source/extensions/filters/http/local_ratelimit/local_ratelimit.cc:96]: It does introduce extra time only useful for debugging, isn't it?

Comment (mattklein123) [reply source/extensions/filters/http/local_ratelimit/local_ratelimit.cc:96]: Yeah I would wrap the entire loop in whether debug logging is enabled or not.

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Thanks generally LGTM. Can you merge main also?

/wait

File: source/extensions/filters/http/local_ratelimit/local_ratelimit.cc (hunk 1 above)
Comment (zirain) [reply source/extensions/filters/http/local_ratelimit/local_ratelimit.cc:96]: sounds good, I will update it later.

Comment (zirain) [reply source/extensions/filters/http/local_ratelimit/local_ratelimit.cc]: updated, PTAL.

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**